bcipy/simulator/README.md

```diff
@@ -43,6 +45,8 @@ For example,
 - `m`: Path to a pickled (.pkl) signal model. One or more models can be provided.
 - `n`: Number of simulation runs
 - `o`: Output directory for all simulation artifacts.
+- `s`: Sampling strategy to use; by default the TargetNonTargetSampler is used. The value provided should be the class name of a Sampler.
+- `sampler_args`: Arguments to pass in to the selected Sampler. Some samplers can be customized with further parameters. These should be structured as a JSON dictionary mapping keys to values. For example: `--sampler_args='{"inquiry_end": 4}'`
```

Comment:
Is this the only parameter that could change in sampling? It may be helpful to have a `custom_args=` that could be used to modify other parameters in the future, too.

Reply:
Any of the sampler arguments can be provided. You can inspect the output from the GUI to see what this looks like. 

In regards to making this more generic, I would like to keep this pretty specific to samplers for now and see how things evolve. I think if there are a number of simulator objects that require more configurability we may want a different solution (such as a sim_config file).